### PR TITLE
Implement cogs

### PR DESCRIPTION
**Pull request by RuiL1904**
### Achievements

- [x] Implemented cogs
- [x] Fixed bug regarding belts attributions
- [x] Created `$load` and `$unload` command in order to load and unload cogs

### Missing features

- [ ] Send private message to the promoted ninja informing about the requirements for the next belt (Solves: #15)

### CODE REVIEW

**Review by danielsp45 — APPROVED** [review]
Everything looks good 👍🏻
Great job!

**Comment by RuiL1904** [comment]
Thanks 🙏🏼 
Can I ask a review from @nelsonmestevao as well?

File: bot/client.py [hunk 1]
```diff
@@ -1,98 +1,38 @@
...
+    client.unload_extension(f'bot.cogs.{extension}')
+    
+    await ctx.send(
+        f'O cog {extension} foi desativado.'
+    )
 
-        await ctx.send(f'{user} agora és cinturão {belt}')
+for filename in os.listdir('bot/cogs'):
+    if filename.endswith('.py'):
+        client.load_extension(f'bot.cogs.{filename[:-3]}')
```
Comment (nelsonmestevao) [inline bot/client.py:37]: I am not a big fan of these dynamic imports from the file system. Is this some kind of standard practice of discord bots?

Comment (RuiL1904) [reply bot/client.py:37]: This was the way I made it on my personal bot. Anyway, I'm opened for new suggestions!

**Pull request merged**